packages/blocks/src/page-block/default/default-page-block.ts

```diff
@@ -481,11 +480,6 @@ export class DefaultPageBlockComponent extends LitElement implements BlockHost {
       this.embedOption = embedOption;
       this.requestUpdate();
     });
-    tryUpdateGroupSize(this.page, 1);
-    this.addEventListener('keydown', e => {
-      if (e.ctrlKey || e.metaKey || e.shiftKey) return;
```

Comment:
I think we shouldn't modify the width and height in default block mode because the zoom is always `1`, and there's no edgeless UI in this mode